plenigo_plugin/PlenigoURLManager.php

```diff
@@ -128,7 +131,7 @@ private function sanitize_url($current_url) {
      * @return string a standardized URL with all its fields
      */
     private function unparse_url($parsed_url) {
-        //unset blank values
+//unset blank values
```

Comment:
Formatted the file when I was having a syntax error, so the formatter flattens everything...